roles/awx/templates/infra/awx-operator.yml

```diff
@@ -0,0 +1,2853 @@
+                    type: object
+                  requests:
+                    properties:
+                      cpu:
+                        type: string
+                      ephemeral-storage:
+                        type: string
+                      memory:
+                        type: string
+                      storage:
+                        type: string
+                    type: object
+                type: object
+              web_tolerations:
+                description: node tolerations for the web pods
+                type: string
+              web_topology_spread_constraints:
+                description: topology rule(s) for the web pods
+                type: string
+            type: object
+          status:
+            properties:
+              URL:
+                description: URL to access the deployed instance
+                type: string
+              adminPasswordSecret:
+                description: Admin password secret name of the deployed instance
+                type: string
+              adminUser:
+                description: Admin user of the deployed instance
+                type: string
+              broadcastWebsocketSecret:
+                description: Broadcast websocket secret name of the deployed instance
+                type: string
+              conditions:
+                description: The resulting conditions when a Service Telemetry is
+                  instantiated
+                items:
+                  properties:
+                    lastTransitionTime:
+                      type: string
+                    reason:
+                      type: string
+                    status:
+                      type: string
+                    type:
+                      type: string
+                  type: object
+                type: array
+              image:
+                description: URL of the image used for the deployed instance
+                type: string
+              migratedFromSecret:
+                description: The secret used for migrating an old instance
+                type: string
+              postgresConfigurationSecret:
+                description: Postgres Configuration secret name of the deployed instance
+                type: string
+              secretKeySecret:
+                description: Secret key secret name of the deployed instance
+                type: string
+              upgradedPostgresVersion:
+                description: Status to indicate that the database has been upgraded
+                  to the version in the status
+                type: string
+              version:
+                description: Version of the deployed instance
+                type: string
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: awx-operator-controller-manager
+  namespace: awx
```

Comment:
The namespaces should be set to the Ansible variable ansible_namespace

Reply:
thanks, it's included now. i updated the documentation to include a reminder for this